src/librustc_errors/lib.rs

```diff
@@ -527,6 +529,7 @@ impl Level {
             Fatal | PhaseFatal | Error => "error",
             Warning => "warning",
             Note => "note",
+            TopLevel => "note",
```

Comment:
Ahhh this is unfortunate, external tools parsing rustc JSON output for example wouldn't be able to tell this apart. Which means unavoidable inconsistency between rustc output and external tools.